mediathread/templates/base.html

```diff
@@ -272,6 +272,11 @@ <h4 id="logo_ccnmtl">
 
                 /* Select from list: Chrome, Firefox, Safari, MSIE */
                 shieldbrowser({"Chrome": 1, "Firefox": 3.6, "Safari": 4, "MSIE": 10 });
+                if (BrowserDetect.browser === 'Safari' &&
+                        window.location.protocol !== 'https:') {
+                    window.location =
+                        window.location.href.replace('http://', 'https://');
+                }
```

Comment:
This looks good, I'll merge when tests pass, but I would also like to look into doing this in django middleware instead of javascript.